src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Project Duke CS2103
+ * Done by Hong Jin.
+ */
+
+package duke;
+
+import duke.task.*;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.time.format.DateTimeFormatter;
+import java.util.*;
+import java.time.LocalDate;
+
+/**
+ * public class Storage to handle file input/read and file output/saving of text file.
+ */
+public class Storage {
+    private String filepath;
+
+    /**
+     * public constructor for Storage.
+     * @param filePath
+     */
+    public Storage(String filePath) {
+        this.filepath = filePath;
+    }
+
+    /**
+     * public method loadfromFile to load/read commands from text file.
+     * @return an arraylist containing the list of task read from file.
+     */
+    public ArrayList<Task> loadFromFile() {
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(" d/MM/uuuu");
+        ArrayList<Task> memo = new ArrayList<>();
```

Comment:
Maybe this attribute should be plural too?

Reply:
oh yes, thanks for pointing that out :)